src/legacy/core_plugins/kibana/public/management/sections/settings/advanced_settings.js

```diff
@@ -170,6 +171,13 @@ export class AdvancedSettings extends Component {
         <EuiSpacer size="m" />
         <CallOuts />
         <EuiSpacer size="m" />
+
+        <AdvancedSettingsVoiceAnnouncement
+          query={query}
```

Comment:
Since `AdvancedSettingsVoiceAnnouncement` uses the query's `text` field only, I think it makes sense to pass just `query.text`
```suggestion
          queryText={query.text}
```